index.d.ts

```diff
@@ -1,5 +1,26 @@
-class PolyK {
-    static Slice(p: number[][][], ax: number, ay: number, bx: number, by: number)
+declare var PolyK: {
+    Slice(p: number[], ax: number, ay: number, bx: number, by: number): number[],
```

Comment:
Started adding JSDocs, this will also help with defining the Typescript definition a bit better.

With that I noticed that the output of Slice is a `number[][]` since it might contain two arrays of polygons.

```javascript
/**
 * Slice
 *
 * @param {number[]} Polygon
 * @param {number} ax Start Coordinate [x]
 * @param {number} ay Start Coordinate [y]
 * @param {number} ax End Coordinate [x]
 * @param {number} ax End Coordinate [y]
 * @returns {number[][]} Array of Polygon
 */
```